src/Microsoft.HttpRepl.Tests/Microsoft.HttpRepl.Tests.csproj

```diff
@@ -7,6 +7,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Moq" Version="4.10.1" />
```

Comment:
Do we need the version here? Shouldn't all the versioning information be in build\PackageVersions.targets?

Reply:
I swear I have removed that three or four times. I must be doing something that is putting it back... Removed again in e3302fb